content/_includes/TNHardwareGuide.md

```diff
@@ -318,29 +321,29 @@ As a courtesy to further limit the motherboard choices, consider the Intelligent
 * Remote virtual media for TrueNAS installation or reinstallation
 
 TrueNAS relies on its web-based user interface (UI), but you might occasionally need console access to make network configuration changes.
-TrueNAS administration and sharing default to a single network interface, which can be challenging when you need to upgrade features like LACP aggregated networking.
-The ideal solution is to have a dedicated subnet to access the TrueNAS web UI, but not all users have this luxury. The occasional visit to the hardware console is necessary for global configuration and even for system recovery.
+TrueNAS administration and sharing use a single network interface by default, which can be challenging when you upgrade features like LACP aggregated networking.
+The ideal solution is to have a dedicated subnet to access the TrueNAS web UI, but not all users have this luxury. The occasional visit to the hardware console is necessary for global configuration and system recovery.
 The latest TrueNAS Mini and R-Series systems ship with full-featured, HTML5-based IPMI support on a dedicated gigabit network interface.
 
 ### Power Supply Units
 
-The top criteria to consider for a power supply unit (or PSU) on a TrueNAS system are its:
+The top criteria to consider for a power supply unit (or PSU) on a TrueNAS system are:
 
-* Power capacity (in watts) for the motherboard and number of drives it must support
+* Power capacity (in watts) for the motherboard and the number of drives it must support
 * Reliability
 * Efficiency rating
 * Relative noise
-* Optional redundancy to keep important systems running if one power supply fails
+* Optional redundancy to keep critical systems running if one power supply fails
 
 Select a PSU rated for the initial and a future load placed on it.
 Have a PSU with adequate power to migrate from a large-capacity chassis to a fully-populated chassis.
 Also, consider a hot-swappable redundant PSU to help guarantee uptime.
 Users on a budget can keep a cold spare PSU to limit their potential downtime to hours rather than days.
-A good, modern PSU is efficient and completely integrates into the IPMI management system to provide real-time fan, temperature, and load information. 
+A good, modern PSU is efficient and integrates into the IPMI management system to provide a real-time fan, temperature, and load information. 
```

Comment:
Does the IPMI provide a real-time fan? or does it provide real-time fan information along with temperature and load information? Not sure this change makes sense for what IPMI provides.